snc.sh

```diff
@@ -223,26 +223,30 @@ create_pvs "${CRC_PV_DIR}" 30
 # https://github.com/openshift/cluster-version-operator/blob/master/docs/dev/clusterversion.md#setting-objects-unmanaged
 ${OC} patch clusterversion version --type json -p "$(cat cvo_override.yaml)"
 
-# Get the pod name associated with cluster-monitoring-operator deployment
```

Comment:
For each of these blocks, I think I'd keep a `# cluster-monitoring-operator` comment. Or maybe this could be a helper func?

Reply:
Regarding the commit log
>  If a user want that specific operator he can just remove it from CVO's override list and all the resources are repopulated.

How to do this *must* be documented in `crc`

Reply:
Yes it should go in the doc side.